### ISSUE TITLE
Support the incoming Material UI v1.0

### ISSUE THREAD

**Issue by HelloEdit** [mbrookes/formsy-material-ui#226]
Hello,

What do you think of creating a new branch to support the beta and the new version of Material UI ?
I just realized that the existing code was incompatible with the current betas 😄 

Anyway, thank for this wrapper !

**Comment by rojobuffalo** [comment]
Good idea. Branch created https://github.com/rojobuffalo/formsy-material-ui/tree/dev/material-1.0

**Comment by HelloEdit** [comment]
Thank for the branch, I'll see if I have time later to update the existing code. Why the code does not use the ES6 class system ?

**Comment by rojobuffalo** [comment]
Doesn't use the ES6 class system because it depends on `Formsy.Mixin`. I don't know if they've put out a replacement for that which would be compatible with ES6 classes. If there is a way to upgrade, I'm open to it.

**Comment by HelloEdit** [comment]
↪️@rojobuffalo

Hello,
Please take a look [here](https://github.com/christianalfoni/formsy-react/blob/master/API.md#formsyhoc)
So, what do you think now ? 😄

**Comment by rojobuffalo** [comment]
👍 Looks like it's possible. I can't make it a personal priority right now, but if you want to run with it, I can answer questions and review PRs.

**Comment by HelloEdit** [comment]
The challenge is interesting, I'll see what I can do !

**Comment by HelloEdit** [comment]
Can you take a look on https://github.com/HelloEdit/formsy-material-ui/tree/feature/ES6 ? I started working on it, tell me what you think! 😄

**Comment by HelloEdit** [comment]
This is the end of my contribution, unfortunately! I created a pull request with the code of the transition to ES6, it is not complete but the hardest was made!

**Comment by maisumbruno** [comment]
@rojobuffalo Is compatible with material-ui@v1-beta? How do I use it?

**Comment by sanathp** [comment]
Waiting for this release
